File: Kody-master/klasa 2/aiter_zadania/przeszukiwanie_liniowe/przeszukiwanie.py
#! /usr/bin/env python
# -*- coding: utf-8 -*-

def main(args):
    n = 6
    szuk = 4
    tab = [1, 5, 3, 8, 0, 6, None]
    tab[n] = szuk
    i = 0
    while tab[i] != szuk:
        i += 1
    if i < n:
        print("Znaleziony")
    else:
        print("Nie znaleziony")
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))

File: Kody-master/klasa 2/aiter_zadania/przeszukiwanie_liniowe/przeszukiwanie.cpp
#include <iostream>

using namespace std;

int main(int argc, char **argv) 
{
    int n = 6;
    int szuk = 7;
    int tab[n + 1] = {1, 5, 3, 8, 0, 6};
    tab[n] = szuk;  // wartownik
    int i = 0;
    while (tab[i] != szuk) i++;
    if (i < n)
        cout << "Znaleziony";
    else
        cout << "Nie znaleziony";
    return 0;
}

